generator.js

```diff
@@ -43,11 +44,21 @@ const processFile = (file, currentPath) => {
 
 const filePath = path.join(__dirname, 'template')
 const projectPath = path.join(process.cwd(), appTitle)
+
 console.log(`Creating ${projectPath} `)
 console.log('Copying over project files')
 processFile('', filePath)
 console.log('Installing NPM Depenedencies')
 execSync('npm install', {cwd: projectPath, stdio: 'inherit'})
+console.log('Initializing a git repository')
+console.log('Making the initial commit !')
+const simplegit = git(projectPath)
+simplegit.init().add('./*').commit('Init Tram-One-App !', (e, d) => {
+  if (e) console.log(e)
+  else {
+    console.log(d)
+  }
```

Comment:
what does the logging usually show? potentially we should have our own strings here depending on what the messaging usually shows. 

If nothing else, this logic could probably be changed to
```
console.log(e || d)
```

Reply:
we should avoid logging the data here. Instead we should just log either `Successfully created commit` or `Failed to create commit`. 

If we fail, that's fine, and I'm not too passionate about doing detailed logging here.